firmware-support/bittide-hal/build.rs

```diff
@@ -31,9 +31,7 @@ fn main() {
 
     let dir = memmap_dir();
 
-    // TODO when this is added it seems to interact weirdly with CI caching
-    //
-    // println!("cargo::rerun-if-changed={}", dir.display());
+    println!("cargo::rerun-if-changed={}", dir.display());
```

Comment:
Oh that works reliably with caching now? Nice :)

Reply:
I believe so :). Let's find out..